components/server/src/auth/verification-service.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { User } from "@gitpod/gitpod-protocol";
+import { log } from "@gitpod/gitpod-protocol/lib/util/logging";
+import { inject, injectable, postConstruct } from "inversify";
+import { Config } from "../config";
+import { Twilio } from "twilio";
+import { ServiceContext } from "twilio/lib/rest/verify/v2/service";
+import { UserDB, WorkspaceDB } from "@gitpod/gitpod-db/lib";
+
+@injectable()
+export class VerificationService {
```

Comment:
Currently this class is part of the community edition, but only bound in the EE container.

As mentioned in GitpodServerImpl, I think it should move to the EE side entirely.